src/main/resources/sql/bridge_participantoptions.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS bridge_participantoptions_<dateSuffix> (
+    healthDataCode      CHAR(36)        ENCODE RAW          NOT NULL,
+    studyKey            VARCHAR(50)     ENCODE BYTEDICT     NOT NULL,
+    data                VARCHAR(500)    ENCODE BYTEDICT     NOT NULL
```

Comment:
It looks like we're also storing email prefs in here:
{\"SHARING_SCOPE\":\"ALL_QUALIFIED_RESEARCHERS\",\"EMAIL_NOTIFICATIONS\":\"false\"}

This is probably fine for now, but it might make it difficult to add more stuff into the participant options in the future.